packages/ra-data-graphql-simple/src/buildVariables.test.ts

```diff
@@ -173,3 +173,181 @@ describe('buildVariables', () => {
         });
     });
 });
+
+describe('buildVariables with meta param', () => {
+    const introspectionResult = {
+        types: [
+            {
+                name: 'PostFilter',
+                inputFields: [{ name: 'tags_some' }],
+            },
+        ],
+    };
+    describe('GET_LIST', () => {
+        it('returns correct variables', () => {
+            const params = {
+                filter: {
+                    ids: ['foo1', 'foo2'],
+                    tags: { id: ['tag1', 'tag2'] },
+                    'author.id': 'author1',
+                    views: 100,
+                },
+                pagination: { page: 10, perPage: 10 },
+                sort: { field: 'sortField', order: 'DESC' },
+                meta: { sparseFields: [] },
```

Comment:
To reduce the scope and test only `meta`, I suggest to simplify the params. Something like:

```suggestion
                filter: {},
                meta: { sparseFields: [] },
```
Same for the other tests